sshd/command.go

```diff
@@ -40,8 +40,13 @@ func execCommand(c *Command, args []string, w StringWriter) error {
 	if c.Flags != nil {
 		fl, fs = c.Flags()
 		if fl != nil {
-			//TODO: handle the error
-			fl.Parse(args)
+			// SetOutput() here in case fl.Parse dumps usage.
+			fl.SetOutput(w.GetWriter())
```

Comment:
so this has `fl` write errors to `w` instead of stderr then?

Reply:
yes